apps/server/src/models/transactions.ts

```diff
@@ -90,8 +97,20 @@ Transactions.init(
       type: DataTypes.STRING,
       allowNull: true,
     },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
```

Comment:
I thought timestamps adds this already... why add it here again?

Reply:
I could not access it  with  merely setting timestamps to 'true' though

Reply:
That's strange